cmd/collector/app/sanitizer/zipkin/span_sanitizer.go

```diff
@@ -121,3 +122,25 @@ func (s *parentIDSanitizer) Sanitize(span *zc.Span) *zc.Span {
 	span.ParentID = nil
 	return span
 }
+
+func NewErrorTagSanitizer() Sanitizer {
+	return &errorTagSanitizer{}
+}
+
+type errorTagSanitizer struct {
+	log spanLogger
+}
+
+func (s *errorTagSanitizer) Sanitize(span *zc.Span) *zc.Span {
+	for i := 0; i < len(span.BinaryAnnotations); i++ {
+		anno := span.BinaryAnnotations[i]
+		if strings.EqualFold("error", anno.Key) {
+			anno.AnnotationType = zc.AnnotationType_BOOL
+			anno.Value = []byte("error")
```

Comment:
https://github.com/openzipkin/zipkin/blob/2755987aa31699bf02d9e6e0366453092ba4a944/zipkin/src/main/java/zipkin/Constants.java#L147

>  the value is a human readable message associated with an error.

I think we should not override the value. There will be an annotation with error key plus bin annotation with an error key and value error message. What we should probably do is:
1. create an error tag for annotation 
2. create a log entry for binary annotation

I am not sure how this would affect older jaeger-ot clients using zipkin model.